docs/account/enterprise.mdx

```diff
@@ -7,95 +7,108 @@ icon: gem
 
 import { calendar } from "/snippets/calendar-link.mdx";
 
-We understand that QA coverage is critical and often needed immediately. That's why we're offering our **4x4 Guarantee**: try TestDriver Enterprise and we guarantee you will see 4 tests in your first 4 weeks!
+TestDriver Enterprise is designed for fast-moving teams that need comprehensive QA coverage for complex applications. Our **4x4 Guarantee** ensures you get 4 custom tests created specifically for your application within your first 4 weeks.
 
-Unlike other QA services that take months and cost hundreds of thousands of dollars, TestDriver gets you up and running in just 30 days.
+We specialize in testing scenarios that other tools can't handle - desktop applications, browser extensions, multi-platform workflows, and complex integrations that require custom infrastructure and expert support.
 
 <Card
-  title="Book a Call"
+  title="Schedule a Consultation"
   icon="calendar"
   href="https://calendly.com/d/cq23-qyn-3v6/testdriver-ai-demo"
   link
 >
-  Don't miss out on the cutting-edge of AI and QA testing!
+  Discuss your testing requirements and custom infrastructure needs.
 </Card>
 
 ---
 
-## Pricing
+## Enterprise Plans
 
-We're confident you'll love TestDriver, which is why we're offering this risk-free trial. After the trial, plans start at just **$995/month**, which includes:
+TestDriver Enterprise plans start at **$995/month** and include:
 
-- **12,500 runner minutes per month**: Enough to run 30 tests **2-5 times per day**.
-- Full access to our enterprise test dashboards.
-- Seamless integration with any CI/CD pipeline.
+- **12,500 runner minutes per month**: Sufficient capacity for continuous testing of your custom test suite.
+- Enterprise-grade test dashboards with advanced analytics.
+- Full CI/CD pipeline integration with custom configurations.
+- Dedicated infrastructure and ongoing support for complex testing scenarios.
```

Comment:
It'll be interesting to see what the economics break down to be.  My napkin math for 12,500 minutes is ~9 days = ~$150.

If the monthly support cost is a few hours, then it's break-even.

If we don't restrict usage, simply doing parallel tests or re-running a suite puts the costs upside-down.

I can see this offering being a base of $10k for managed infra.  I was paying $4k/mo 10 years ago for managing ~12 LAMP servers.  Closer to $8-10k with managed MYSQL too.